webapp/view/ListReport.view.xml

```diff
@@ -0,0 +1,187 @@
+<mvc:View
+	xmlns="sap.m"
+	xmlns:mvc="sap.ui.core.mvc"
+	xmlns:f="sap.f"
+	xmlns:core="sap.ui.core"
+	xmlns:fb="sap.ui.comp.filterbar"
+	controllerName="products.app.controller.ListReport">
+
+		<f:DynamicPage 
+			id="dynamicPageId" 
+			preserveHeaderStateOnScroll="true" 
+			showFooter="true" 
+			fitContent="true">
+			<!-- DynamicPage Title -->
+			<f:title>
+				<f:DynamicPageTitle>
+					<f:heading>
+						<Title text="{i18n>ListReportTitle}"/>
+					</f:heading>
+					
+				</f:DynamicPageTitle>
+			</f:title>
+
+			<!-- DynamicPage Header -->
+			<f:header>
+				<f:DynamicPageHeader>
+					
+					<f:content>
+						
+						<fb:FilterBar id="filterbar" persistencyKey="myPersKey" useToolbar="false" >
+							<fb:filterGroupItems >
+
+								<fb:FilterGroupItem 
+									name="search" 
+									label="{i18n>NameText} {i18n>AndText} {i18n>DescriptionText}" 
```

Comment:
This is not quite great because people from Western Asia could use your application. They read right to left, not left to right like most of the world. So you will need to use placeholders and templates of i18n. 

Tomorrow I will show you how to do it.

Reply:
Placeholder will change app from 'left to right' to 'right to left' so people from Western Asia could use your apps